models/users/scratch/snowplow_web_users_this_run.sql

```diff
@@ -23,25 +23,38 @@ select
 
   -- first page fields
   a.first_page_title,
-
   a.first_page_url,
-
   a.first_page_urlscheme,
   a.first_page_urlhost,
   a.first_page_urlpath,
   a.first_page_urlquery,
   a.first_page_urlfragment,
 
-  c.last_page_title,
+  a.geo_country as first_geo_country,
```

Comment:
Doesn't the renaming of these columns pose problems in the form of breaking changes as well? Not here but downstream for `snowplow_web_users`?

Reply:
We didn't previous have this in users at all, so we are adding it new in there, so no breaking changes as all new columns